src/toolkit.cpp

```diff
@@ -1165,7 +1165,7 @@ std::string Toolkit::GetElementAttr(const std::string &xmlId)
     }
     // If not found at all
     if (!element) {
-        LogInfo("Element with id '%s' could not be found", xmlId.c_str());
+        LogInfo("Element '%s' not found", xmlId.c_str());
```

Comment:
The original message seems to be a bit clearer?

Reply:
Why should this message be different than all the others?

Reply:
Why can't it be different? Unless you're filtering on that specific string, does it matter if they're different?